### PR TITLE
orloj 0.10: rewrite from "andark"

### PR DESCRIPTION
**Pull request by pavelmachek**
Change the look from (copying from andark), while still providing sunset time and sun position.

### CODE REVIEW

File: apps/orloj/ChangeLog [hunk 1]
```diff
@@ -1,3 +1,4 @@
 0.01: attempt to import
 0.02: Minor code improvements
 0.03: Minor code improvements
+0.10: Restart, sttart from "andark" adding astronomical features to it
```
Comment (bobrippling) [inline apps/orloj/ChangeLog:4]: Minor typo here :)

```suggestion
0.10: Restart, start from "andark" adding astronomical features to it
```

**Comment by bobrippling** [comment]
Looks good, happy to merge with the little fix

**Comment by pavelmachek** [comment]
Fixed, thanks for review.

**Comment by bobrippling** [comment]
You're welcome - thanks for the PR

**Pull request merged**